sdk/src/compute_budget_processor.rs

```diff
@@ -0,0 +1,152 @@
+//! Process compute_budget instructions outside of program-runtime to
```

Comment:
mostly just moved `process_instructions(...)` from program-runtime to sdk to reduce dependency.